### DISCUSSION TITLE
how to set my name and save it?

### DISCUSSION THREAD

**Discussion by TheNoobCode135** [Prodigy-Hacking/ProdigyMathGameHacking#855] [(A) Anything]
I want to save my set name
use "set name(client side only)", like I want to set my name to Dark Dragon when I typed and press ok, I reload the web, but my name didn't change :/ can someone help

**Comment by PatheticMustan** [comment]
Other players will not see it.... are you sure you want this feature?

**Comment by LeoBadeaux** [comment]
Using the option on the cheat menu, you can change your name, but it's limited to the names that prodigy has allowed, and it will show to everybody.
